core/trino-main/src/main/resources/webapp/src/components/LivePlan.jsx

```diff
@@ -181,7 +181,7 @@ export class LivePlan extends React.Component<LivePlanProps, LivePlanState> {
         }
     }
 
-    refreshLoop() {
+    refreshLoop = () => {
```

Comment:
the added types and other changes like this one, are they strictly related to flow updated, or collateral?

Reply:
related to flow update which validates if binding is correct